sdk/tools/cabman/CMakeLists.txt

```diff
@@ -1,11 +1,20 @@
 
+#add_definitions(-DDBG)
+
 list(APPEND SOURCE
     cabinet.cxx
+    cabinet.h
```

Comment:
Aren't headers included automatically?

Reply:
yes but now I can see them in visual studio ;)